assessments/0006-gateway-api.md

```diff
@@ -0,0 +1,240 @@
+# Assessment: Project Kubernetes Gateway API
+
+Prepared by: Meha Bhalodiya ([@mehabhalodiya](https://github.com/mehabhalodiya))<br>
+Date: 2021-03-03
+
+## Introduction
+
+This document assesses the quality and completeness of the Kubernetes Gateway API [project's](https://github.com/kubernetes-sigs/gateway-api) documentation and [website](https://gateway-api.sigs.k8s.io/).
+
+This document:
+
+- Measures existing documentation quality against the CNCF’s standards
+- Recommends specific and general improvements
+- Provide examples of great documentation as a reference
+- Identifies key improvements with the largest return on investment
+
+
+## How this document works
+
+The assessment is divided into three sections:
+
+- **Project documentation:** for end-users of the project; aimed at people who intend to use it
+- **Contributor documentation:** for new and existing contributors to the project
+- **Website:** branding, website structure, and maintainability
+
+Each section rates content based on different [criteria](criteria.md).
+
+
+## Project documentation
+
+| Criteria                   | 1   | 2   | 3   | 4   | 5   |
+| ---                        | --- | --- | --- | --- | --- |
+| Information architecture   |     |  ✅  |     |     |     |
+| New user content           |     |     |  ✅  |     |     |
+| Content maintainability & site mechanics   |     |     |     |  ✅  |     |
+| Content creation processes |     |     |     |  ✅  |     |
+
+
+Scale:
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+
+### Comments
+
+**Information architecture**:
+
+The brief information is mentioned on [the Introductory](https://gateway-api.sigs.k8s.io/) page, but we need to organize it well. Also, [API conventions](https://gateway-api.sigs.k8s.io/concepts/guidelines/?h=api+conve#api-conventions) are mentioned in brief, great work!
+
+**New user content**: 
+
+We have a great, clearly labelled [Getting Started](https://gateway-api.sigs.k8s.io/v1alpha2/guides/getting-started/) page, which is awesome! However, the getting started guide is fairly high level and doesn't answer some of the following questions:
+- What are the prerequisites that need to be installed beforehand? Or What else do I need aside from Gateway Controller?
+* Where to go and what to do after reading the “Getting Started” guide?
+
+**Content maintainability**:
+
+Our documentation is searchable, which is great! Also, we have a clearly documented method for [versioning](https://gateway-api.sigs.k8s.io/references/releases/#versioning) our content.
```

Comment:
This doesn't match the discussion we had about versioning (and I don't think it matches the latest copy in the google document)

Reply:
Sorry, was confused -- but I think this comment conflicts with what we're suggesting below about versioning the documentation.

Reply:
So should I write:

> Apart from this, we need to work on the documented method for [versioning](https://gateway-api.sigs.k8s.io/references/releases/#versioning) our content.

instead of:

> Also, we have a clearly documented method for [versioning](https://gateway-api.sigs.k8s.io/references/releases/#versioning) our content.

?

Reply:
Oh yeah, I am removing it, would be better!